Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -233,7 +237,7 @@ private func validateRuleIdentifiers(configuredRules: [Rule], disabledRules: [St
     return validRuleIdentifiers
 }
 
-private func containsDuplicatedRuleIdentifiers(_ validDisabledRules: [String]) -> Bool {
+private func containsDuplicatedRuleIdentifiers(validDisabledRules: [String]) -> Bool {
```

Comment:
There's nothing specific to the IDs being _disabled_ in this function, so I'll just change to `containsDuplicateIdentifiers(_ identifiers: [String])`.